_data/navigation.yml

```diff
@@ -190,6 +190,8 @@ setup:
             url: /setup/features/user-data/
           - title: Configure the Script stage
             url: /setup/features/script-stage/
+      - title: Use External Account Configuration
+        url: /setup/configuration/
```

Comment:
I think this needs to go under /setup/other_config/
Maybe /setup/other_config/external?